src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -28,24 +30,91 @@ class CompanyStep extends React.Component {
             companyName: lodashGet(props, ['achData', 'companyName'], ''),
             addressStreet: lodashGet(props, ['achData', 'addressStreet'], ''),
             addressCity: lodashGet(props, ['achData', 'addressCity'], ''),
-            addressState: lodashGet(props, ['achData', 'addressState'], ''),
+            addressState: lodashGet(props, ['achData', 'addressState']) || 'AK',
             addressZipCode: lodashGet(props, ['achData', 'addressZipCode'], ''),
             companyPhone: lodashGet(props, ['achData', 'companyPhone'], ''),
             website: lodashGet(props, ['achData', 'website'], ''),
             companyTaxID: lodashGet(props, ['achData', 'companyTaxID'], ''),
             incorporationType: lodashGet(props, ['achData', 'incorporationType'], ''),
             incorporationDate: lodashGet(props, ['achData', 'incorporationDate'], ''),
-            incorporationState: lodashGet(props, ['achData', 'incorporationState'], ''),
+            incorporationState: lodashGet(props, ['achData', 'incorporationState']) || 'AK',
             industryCode: lodashGet(props, ['achData', 'industryCode'], ''),
             hasNoConnectionToCannabis: lodashGet(props, ['achData', 'hasNoConnectionToCannabis'], false),
             password: '',
         };
     }
 
+    /**
+     * Validating that this is a valid address (PO boxes are not allowed)
+     *
+     * @param {String} value
+     * @returns {Boolean}
+     */
+    isValidAddress(value) {
```

Comment:
I would have put some of these functions in a new lib: `src/libs/ValidationUtils.js`